cfme/tests/configure/test_default_views.py

```diff
@@ -2,15 +2,18 @@
 
 import pytest
 import re
+from functools import partial
 from cfme.fixtures import pytest_selenium as sel
 import cfme.web_ui.toolbar as tb
-from cfme.web_ui import ButtonGroup, form_buttons
+from cfme.web_ui import ButtonGroup, form_buttons, Quadicon
 from utils.conf import cfme_data
 from utils.providers import setup_a_provider as _setup_a_provider
 from cfme.configure import settings  # NOQA
 from cfme.services.catalogs import catalog_item  # NOQA
 from cfme.services import workloads  # NOQA
 
+cfg_btn = partial(tb.select, 'Configuration')
```

Comment:
I appreciate the need to locate this during a test, but I think it should be part of the descriptor in `exp_comp`
```yaml
    exp_comp:
            infra:
                - Compare/infrastructure_hosts/Configutation/Compare Selected Hosts
```

Then, in the code that uses it,
`tb.select(name[2]. name[3])`